src/main/java/chess/ChessGame.java

```diff
@@ -1,31 +1,49 @@
 package chess;
 
+import static chess.domain.Command.STATUS;
+import static chess.domain.GameStatus.GAME_OVER;
+import static chess.domain.GameStatus.WHITE_TURN;
+
 import chess.domain.Command;
+import chess.domain.GameStatus;
 import chess.domain.chessboard.ChessBoard;
+import chess.domain.chessboard.ChessBoardGenerator;
 import chess.domain.position.Position;
+import chess.util.RetryUtil;
 import chess.view.InputView;
 import chess.view.OutputView;
 import java.util.List;
 
 public class ChessGame {
+    private GameStatus gameStatus;
+
     public void run() {
         OutputView.printStartMessage();
-        Command command = Command.getStartCommand(InputView.readCommand());
-        ChessBoard chessBoard = ChessBoard.initializeChessBoard();
-        while (!command.isEnd()) {
+        Command command = RetryUtil.read(() -> Command.getStartCommand(InputView.readCommand()));
```

Comment:
```suggestion
        RetryUtil.read(() -> Command.getStartCommand(InputView.readCommand()));
```
Command가 선언되어야할 필요가 있을까요?